### DISCUSSION TITLE
What's the best way to start contributing to the code?

### DISCUSSION THREAD

**Discussion by joglomedia** [vitodeploy/vito#313] [Q&A]
Hi @saeedvaziry 

After a few days of reviewing and exploring Vito, I'm eager to contribute. However, I'm still unsure where to begin. Has the v2 migration been completed, so we can focus on improving existing features and adding new ones?

**Comment by saeedvaziry** [comment]
@joglomedia migration to filament is almost done. I am migrating the tests which is also almost finished. will free up some issues very soon.
